index.rst

```diff
@@ -1,40 +1,378 @@
+- to announce a new technology, serving as a high-level overview complementing software documentation,
+- to propose an architecture, possibly becoming the subject of a request for comment (RFC).
 
-Write the docs; right here.
+Technote Platform Design Choices
+=================================
+
+Although technotes are still in an early, minimally-viable state, (see :ref:`implementation`), they also demonstrate several design decisions that characterize the platform.
+
+Technotes are native to the web
+-------------------------------
+
+Although a preponderance of astronomy literature is published as PDFs, and many management documents as Word documents, those formats are archaic solutions from an era when documents were read in print.
+Modern HTML, CSS and JavaScript allow for better, more innovative, and frictionless reading experiences than PDFs and Word documents can provide.
+Modern readers expect to be able to search for content, click a link, and read it immediately in the browser.
+
+Technotes are written in reStructuredText
+-----------------------------------------
+
+A fundamental requirement of the technote platform is that documents are written in a plain text format, and are then compiled into a website. This way, content is written in common text editors and hosted on GitHub for collaboration (the same workflow we already use for source code).
+Many plain text formats have been invented for writing, however, not all formats are ideal for technotes.
+For example, LaTeX, though common in astronomy, is tied too closely to PDF output and makes little sense for a web-native syntax.
+`Markdown <http://daringfireball.net/projects/markdown/>`_ is a hugely popular format among developers, in part because of its intuitive simplicity.
+However, this simplicity has prompted developers to `add non-standard extensions to the format <http://commonmark.org>`_ to their markup to add semantics and style to content.
+
+Instead, reStructuredText is an ideal markup format for technotes.
+reStructuredText's syntax and build process was designed to be extended, and in fact, those extensions are made in DM's primary language, Python.
+Because the Python community has adopted reStructuredText as its official markup language, a our technote platform can take advantage of excellent open source tools, such as the Sphinx build system and the Read the Docs documentation hosting site.
+For these same reasons, reStructuredText and the Sphinx build toolchain are also used by DM's new software documentation platform.
+This allows both documentation platforms to benefit from the same bespoke infrastructure development, such as SQuaRE's `documenteer`_ package.
+
+Technotes are single-page documents
+-----------------------------------
+
+This limitation is made purposefully to limit the scope of technotes, make it possible to easily print a technote in its entirely, and allow readers to use their browser's search feature to navigate a document.
```

Comment:
Although, ironically, if you are talking about printing you had better be able to format it in PDF properly. Nothing worse than printing an HTML page and seeing it split a line in half horizontally across pages.